packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-credit-button-strategy.ts

```diff
@@ -98,6 +124,183 @@ export default class PaypalCommerceCreditButtonStrategy implements CheckoutButto
         }
     }
 
+    private async _onCancel() {
+        const state = this._store.getState();
+        const billingAddress = state.billingAddress.getBillingAddressOrThrow();
+        const resetAddress = this._resetAddress(billingAddress);
+        await this._store.dispatch(this._billingAddressActionCreator.updateAddress(resetAddress));
+        await this._store.dispatch(this._consignmentActionCreator.updateAddress(resetAddress));
+        await this._updateOrder();
+    }
```

Comment:
Do we need to wrap them in a try catch, in case the calls error out?

Reply:
We don't need. Each method have error handling inside